src/main/java/platform/codingnomads/co/springweb/resttemplate/GET/getForObject/GetForObjectDemo.java

```diff
@@ -24,13 +28,35 @@ public static void main(String[] args) {
     @Bean
     public CommandLineRunner run() throws Exception {
         return args -> {
-
+            // Quotes
             QuoteTemplate[] randomQuote;
             randomQuote = restTemplate.getForObject("https://zenquotes.io/api/random/", QuoteTemplate[].class);
             System.out.println(Arrays.toString(randomQuote));
 
+            // Excuses
+            ExcuseTemplate[] randomExcuse;
+            randomExcuse = restTemplate.getForObject("https://excuser-three.vercel.app/v1/excuse/party/2", ExcuseTemplate[].class);
+            System.out.println(Arrays.toString(randomExcuse));
+
+            // Bored
+            BoredTemplate randomActivity;
+            randomActivity = restTemplate.getForObject("http://www.boredapi.com/api/activity?participants=1", BoredTemplate.class);
+            System.out.println(randomActivity);
+
+            // Pass parameters with Map
+            HashMap<String, Integer> boredApiParameters = new HashMap<String, Integer>();
```

Comment:
Since Java 7 you can skip repeating the content of `<>` operator on the right part of the assignment. The compiler will use the one from the left part. If the right part is different, you still need to write it explicitly.

```suggestion
            HashMap<String, Integer> boredApiParameters = new HashMap<>();
```